.github/workflows/periodic.yml

```diff
@@ -13,10 +13,21 @@ jobs:
       - name: Create k8s Kind Cluster
         uses: container-tools/kind-action@v2
         with:
-          cluster_name: kfp-tekton
+          cluster_name: kfp-functional-test
           kubectl_version: v1.29.2
           version: v0.22.0
           node_image: kindest/node:v1.29.2
+      - name: Deploy kfp
```

Comment:
One more thing @tmvfb. Sorry.
Can you please use the new `kfp-cluster` action here? Like what is done in https://github.com/kubeflow/pipelines/blob/2fb4922fc0ecf78f7c8050111171333abdd2812a/.github/workflows/e2e-test.yaml#L26.

Reply:
@hbelmiro, no problem, edited commits and tested [here](https://github.com/prokube-ai/pipelines/actions/runs/9759012845/job/26934822612)